Moonshot/default_bus_layout.tres

```diff
@@ -1,13 +1,6 @@
-[gd_resource type="AudioBusLayout" load_steps=5 format=2]
+[gd_resource type="AudioBusLayout" load_steps=4 format=2]
 
-[sub_resource type="AudioEffectReverb" id=4]
-resource_name = "Reverb"
```

Comment:
Just removed a reverb somehow added to the master audio bus.

Reply:
Oops that was me xD I think we need a small amount of reverb on the master bus - makes all the sounds sound like they are being made in the same room.